src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
+        Lottery lottery = new Lottery();
+        System.out.println(lottery.getRandomBall().toString());
+        System.out.println(lottery.getRandomBall().toString());
+        System.out.println(lottery.getRandomBall().toString());
         // create three balls using class Lottery and print information about them in console
```

Comment:
```suggestion
        for (int i = 0; i < 3; i++) {
            System.out.println(lottery.getRandomBall());
        }
```
remove comments
checklist: 
![image](https://user-images.githubusercontent.com/81781479/186181757-cbdb48ad-a0d0-49b0-99fb-b34585bb0c1d.png)